autorest/azure/cli/token.go

```diff
@@ -112,3 +116,55 @@ func LoadTokens(path string) ([]Token, error) {
 
 	return tokens, nil
 }
+
+// GetTokenFromCli gets a token using Azure CLI 2.0 for local development scenarios.
+func GetTokenFromCli(resource string) (*Token, error) {
+	// This is the path that a develop can set to tell this class what the install path for Azure CLI is.
+	const azureCliPath = "AzureCLIPath"
```

Comment:
To be consistent with the other env vars this value should be `AZURE_CLI_PATH`.

Reply:
The .NET SDK uses the `AzureCLIPath` env var, but I'd be happy to change this. Should I?

Reply:
I see, I wasn't aware of that.  Let's keep it as-is so it's consistent across the SDKs.